src/components/cards/cards.css

```diff
@@ -103,7 +103,7 @@
     color : var(--ca-primary-color)
 }
 .card-vertical {
-    max-width : 30% ;
+    max-width : 300px ;
```

Comment:
use rems instead of pixels.

Reply:
so actually this vertical card i want to have a standard size